internal/pkg/model/schema.go

```diff
@@ -190,6 +190,9 @@ type Agent struct {
 	// Date/time the Elastic Agent was last upgraded
 	UpgradedAt string `json:"upgraded_at,omitempty"`
 
+	// Upgrade status
+	UpgradeStatus string `json:"upgrade_status,omitempty"`
```

Comment:
This file is generated from `model/schema.json`; you should update that and run the json schema generator in order to update the go definition.

Reply:
Do you know how can I run the generator?

Reply:
updated